Tasks/AzurePowerShellV4/TryMakingModuleAvailable.ps1

```diff
@@ -0,0 +1,78 @@
+[CmdletBinding()]
+param (
+    [string]
+    $targetVersion,
```

Comment:
we need to come up with logic where $targetVersion is one of from currently supported hosted agent

Reply:
what is logic for "latest" version

Reply:
> what is logic for "latest" version

We just skip the download flow and in the existing code we pick the latest available as folder.

Reply:
> we need to come up with logic where $targetVersion is one of from currently supported hosted agent

May be we can discuss it later and not make it as part of the current PR.